tests/subsys/llext/simple/prj.conf

```diff
@@ -2,7 +2,7 @@ CONFIG_ZTEST=y
 CONFIG_ZTEST_STACK_SIZE=4096
 CONFIG_LOG=y
 CONFIG_LLEXT=y
-CONFIG_LLEXT_HEAP_SIZE=16
+CONFIG_LLEXT_HEAP_SIZE=32
```

Comment:
This is really my only concern, it will likely cause a mismatch in hardware memory available vs test requirements

Reply:
The testsuite has a `min_ram: 128` entry, I would expect it to exclude any hardware with too little memory :)

Reply:
If necessary I could also add a test-specific CONFIG; that much memory is really only useful on the MPU-enabled tests due to aligment requirements. 
Do you have some target HW in mind where 32k is an issue?